web/services/hosts_next.go

```diff
@@ -1,17 +1,26 @@
 package services
 
 import (
+	"time"
+
 	"github.com/lib/pq"
+	"github.com/trento-project/trento/internal"
 	"github.com/trento-project/trento/web/entities"
 	"github.com/trento-project/trento/web/models"
 	"gorm.io/gorm"
+	"gorm.io/gorm/clause"
 )
 
+const HeartbeatTreshold = 5 * 2 * time.Second
```

Comment:
I thought about this but I couldn't find a place for the constant.

Reply:
`./internal/constants.go`
`./internal/constants/heartbeat.go`
`./internal/heartbeat.go`

I'd rather improperly use `internal` for this (as we are already currently doing for several stuff) instead of duplicating the thing, which would be really easy to overlook or forget imho

Reply:
Alright! Will think about a name and a place.

Reply:
I'm going to address this in #444